package.json

```diff
@@ -62,15 +63,17 @@
     "ws": "^7.3.0"
   },
   "devDependencies": {
-    "@rollup/plugin-json": "^4.1.0",
     "@babel/plugin-transform-runtime": "^7.10.4",
     "@babel/preset-env": "^7.10.4",
+    "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^8.1.0",
+    "css-loader": "^3.6.0",
     "electron": "^9.0.5",
     "electron-builder": "^22.7.0",
     "rollup": "^2.18.1",
     "rollup-plugin-babel": "^4.4.0",
     "rollup-plugin-commonjs": "^10.1.0",
-    "rollup-plugin-terser": "^6.1.0"
+    "rollup-plugin-terser": "^6.1.0",
+    "style-loader": "^1.2.1"
```

Comment:
We don't need `style-loader`. The correct way to add styles I'll discuss in a later comment.